H/Htop/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder
 
 name = "Htop"
-version = v"2.2.0"
+version = v"3.1.2"
 
 # Collection of sources required to complete build
 sources = [
-    "http://hisham.hm/htop/releases/2.2.0/htop-2.2.0.tar.gz" =>
-    "d9d6826f10ce3887950d709b53ee1d8c1849a70fa38e91d5896ad8cbc6ba3c57",
+    ArchiveSource("https://github.com/htop-dev/htop/releases/download/$(version)/htop-$(version).tar.xz",
+    "884bce5b58cb113127860b9e368609019e92416a81550fdf0752052f3a64b388")
```

Comment:
Indentation wouldn't hurt
```suggestion
    ArchiveSource("https://github.com/htop-dev/htop/releases/download/$(version)/htop-$(version).tar.xz",
                            "884bce5b58cb113127860b9e368609019e92416a81550fdf0752052f3a64b388"),
```

Reply:
That was too much indentation.  Not sure what's wrong with the suggestion box